tooltester/README.md

```diff
@@ -0,0 +1,70 @@
+Last tested with ansible 2.2.1.0
+# Master Setup:
+1.  Install the Jenkins pipeline suite
+2.  Install all plugins filtered with "blue ocean"
+3.  Create a pipeline called PipelineTest
+4.  Copy the contents of resources/PipelineTest.groovy into the Pipeline Script textbox
+5.  Check the checkbox:  "This project is parameterized"
+6.  Create the String Parameters mentioned in the [constructParameterMap](https://github.com/ga4gh/dockstore-support/blob/develop/tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java#L609) function. These string parameters are used to configure the Jenkins pipelines to run the correct tool/workflow, versions, etc.
+7.  Install ansible
```

Comment:
note the ansible version you tested with, ansible moves quickly

Reply:
It's mentioned at the top